src/less/components/payex/progress-step.less

```diff
@@ -0,0 +1,45 @@
+.steps {
+    ol {
+        li {
+            a {
+                .steps-number {
+                    &:hover {
+                        color: @white;
+                    }
```

Comment:
Might need to be on li instead

Reply:
moved to li but got problems with text under a-tag. Designer from Payex decided to go with hover effects without the color white.